worker/src/main/java/gov/cms/ab2d/worker/service/WorkerServiceImpl.java

```diff
@@ -16,17 +21,37 @@
 
     private final JobPreProcessor jobPreprocessor;
     private final JobProcessor jobProcessor;
+    private final ShutDownService shutDownService;
+
+    private static List<String> activeJobs = Collections.synchronizedList(new ArrayList<>());
```

Comment:
Does it need to be `static`? I _think_ the Worker bean is a singleton, but even if it is not, each instance would maintain the jobs it is processing anyway.

Reply:
done